code/modules/surgery/organs/tongue.dm

```diff
@@ -41,6 +44,12 @@
 	..()
 	if(say_mod && M.dna && M.dna.species)
 		M.dna.species.say_mod = say_mod
+	if(ask_mod && M.dna && M.dna.species)
+		M.dna.species.ask_mod = ask_mod
+	if(exclaim_mod && M.dna && M.dna.species)
+		M.dna.species.exclaim_mod = exclaim_mod
+	if(yell_mod && M.dna && M.dna.species)
+		M.dna.species.yell_mod = yell_mod
```

Comment:
Bunch all of these together under a single `if(M.dna && M.dna.species)` branch.
Then undo the changes to the species datum, that this code does, in `Remove()`.